platforms/r3-corda-ent/charts/auth/templates/pvc.yaml

```diff
@@ -0,0 +1,47 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Values.prefix }}-auth-etc
+  namespace: {{ .Values.metadata.namespace }}
+spec:
+  accessModes:
+    - ReadWriteOnce
+  storage.nameName: {{ .Values.storage.name }}
```

Comment:
storage.nameName should be storageClassName

Reply:
fixed